release.json

```diff
@@ -7,8 +7,8 @@
     },
     "nightly": {
         "INTEGRATIONS_CORE_VERSION": "master",
-        "OMNIBUS_SOFTWARE_VERSION": "fcd374ae322102d4fcd91a0e7dd4ec5726cdcfb7",
-        "OMNIBUS_RUBY_VERSION": "40bf4965ea054d0967a7cb93ddc1681225df1b4f",
+        "OMNIBUS_SOFTWARE_VERSION": "master",
+        "OMNIBUS_RUBY_VERSION": "chouquette/fix_oci_packaging",
```

Comment:
🐒 Nitpick :
Reminder to swap back to non-dev branch

Reply:
Done :)